src/v/kafka/server/handlers/configs/config_utils.h

```diff
@@ -199,4 +199,94 @@ ss::future<std::vector<R>> unsupported_broker_configuration(
     return ss::make_ready_future<std::vector<R>>(std::move(responses));
 }
 
+template<typename T>
+requires requires(const T& value, const ss::sstring& str) {
+    { boost::lexical_cast<T>(str) } -> std::convertible_to<T>;
+}
+void parse_and_set_optional(
+  cluster::property_update<std::optional<T>>& property,
+  const std::optional<ss::sstring>& value,
+  config_resource_operation op) {
```

Comment:
so this version of parse_and_set_optional now handles the previous 2 use cases (looks like one with and one without the last op parameter)?

Reply:
for the non incremental version the operation is always `set`